src/components/DuosHeader.js

```diff
@@ -75,6 +191,19 @@
       isDataOwner = currentUser.isDataOwner;
     }
 
+    const dropdownLinks = {
+      statistics: {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6963768)
```suggestion
        "Votes Statistics": {
```